it/src/test/java/apoc/it/core/TriggerEnterpriseFeaturesTest.java

```diff
@@ -46,10 +49,15 @@ public class TriggerEnterpriseFeaturesTest {
 
     @BeforeClass
     public static void beforeAll() {
+        final String cypherInitializer = String.format("%s.%s.0",
+                APOC_CONFIG_INITIALIZER, SYSTEM_DATABASE_NAME);
+        final String createInitDb = String.format("CREATE DATABASE %s IF NOT EXISTS", INIT_DB);
+
         // We build the project, the artifact will be placed into ./build/libs
         neo4jContainer = createEnterpriseDB(List.of(TestContainerUtil.ApocPackage.CORE), true)
                 .withEnv(APOC_TRIGGER_ENABLED, "true")
-                .withEnv(TRIGGER_REFRESH, String.valueOf(TRIGGER_DEFAULT_REFRESH));
+                .withEnv(TRIGGER_REFRESH, String.valueOf(TRIGGER_DEFAULT_REFRESH))
+                .withEnv(cypherInitializer, createInitDb);
```

Comment:
Why do we need this?

Reply:
Sorry, I had put some extra code that I probably mistakenly deleted after a rebase :X
It's to check that even if I create a db via `apoc.initializer.*` and then delete it, the implementation still works.
See the test `TriggerEnterpriseFeaturesTest.testDeleteTriggersAfterDatabaseDeletion` (line 193)